kube_apiserver_metrics/assets/dashboards/overview.json

```
@@ -0,0 +1,809 @@
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "time": {},
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "formula": "query1"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "query": "sum:kube_apiserver.grpc_client_handled_total{$cluster_name} by {grpc_code}.as_rate()",
+                      "data_source": "metrics",
+                      "name": "query1"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ],
+              "yaxis": {
+                "include_zero": true,
+                "scale": "linear",
+                "label": "",
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 4,
+              "height": 2
+            }
+          },
+          {
+            "id": 1050701067351742,
+            "definition": {
+              "title": "By (service, method) per second",
+              "title_size": "16",
+              "title_align": "left",
+              "time": {},
+              "type": "toplist",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "formula": "query1",
+                      "limit": {
+                        "count": 10,
+                        "order": "desc"
+                      }
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "query": "sum:kube_apiserver.grpc_client_handled_total{$cluster_name} by {grpc_service,grpc_method}.as_rate()",
+                      "data_source": "metrics",
+                      "name": "query1",
+                      "aggregator": "last"
+                    }
+                  ],
+                  "response_format": "scalar"
+                }
+              ]
+            },
+            "layout": {
+              "x": 4,
+              "y": 0,
+              "width": 4,
+              "height": 2
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 8,
+        "height": 3
+      }
+    }
+  ],
+  "template_variables": [
+    {
+      "name": "cluster_name",
+      "default": "*",
+      "prefix": "cluster_name"
```

we should replace `cluster_name` in favour of `kube_cluster_name` because `cluster_name` is "deprecated" since several other integration are using it and so it can generate tag conflit.

```suggestion
      "name": "kube_cluster_name",
      "default": "*",
      "prefix": "kube_cluster_name"
```

---

Added in new commit https://github.com/DataDog/integrations-core/pull/9757/commits/d3be98b0ce419a980e4898b357a24ebb6e3db486 (will squash later)